x/ibc-account/keeper/handshake.go

```diff
@@ -5,10 +5,7 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
 	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
-	porttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/05-port/types"
 	host "github.com/cosmos/cosmos-sdk/x/ibc/core/24-host"
-
-	"github.com/cosmos/interchain-accounts/x/ibc-account/types"
 )
 
 func (k Keeper) OnChanOpenInit(
```

Comment:
These functions should also error if the active channel is already set

Reply:
That is a good point